en/power_module/cuav_hv_pm.md

```diff
@@ -0,0 +1,28 @@
+# CUAV HV PM(High voltage power module)
```

Comment:
@cuhome What is the connector on the PM? Does it come with cables to connect to flight controllers - which ones?

What flight controllers do you recommend this for/know it will work with? i.e. I am sure it is designed for the CUAV v5+? Have you tested it on others?